tests/framework/e2e/etcd_process.go

```diff
@@ -253,6 +258,62 @@ func (ep *EtcdServerProcess) Close() error {
 	return nil
 }
 
+func (ep *EtcdServerProcess) VerifySchemaVersion(lg *zap.Logger) error {
+	currentEtcdVer, err := ep.getEtcdVersion(lg)
+	if err != nil {
+		return err
+	}
+	prevEtcdVer := &semver.Version{Major: currentEtcdVer.Major, Minor: currentEtcdVer.Minor - 1}
+
+	dbPath := datadir.ToBackendFileName(ep.cfg.DataDirPath)
+	be := backend.NewDefaultBackend(lg, dbPath)
+	defer be.Close()
+	ver, err := schema.UnsafeDetectSchemaVersion(lg, be.BatchTx())
+	if err != nil {
+		return err
+	}
+
+	// in a mix version cluster, the storage version would be set to the cluster version,
+	// which could be lower than the server version by 1 minor version.
+	if currentEtcdVer.LessThan(ver) || ver.LessThan(*prevEtcdVer) {
+		return fmt.Errorf("expect backend schema version to be between [%s, %s], but got %s", prevEtcdVer.String(), currentEtcdVer.String(), ver.String())
+	}
+	if ep.cfg.ExecPath == BinPath.Etcd {
+		return nil
+	}
+	nextVer := semver.Version{Major: currentEtcdVer.Major, Minor: currentEtcdVer.Minor + 1}
```

Comment:
keep the naming consistent. Or remove `Etcd` from all the names
```suggestion
	nextEtcdVer := semver.Version{Major: currentEtcdVer.Major, Minor: currentEtcdVer.Minor + 1}
```